README.md

```diff
@@ -13,7 +13,7 @@
         * [AWS IoT Device Shadow](#aws-iot-device-shadow)
         * [AWS IoT Jobs](#aws-iot-jobs)
         * [AWS IoT Device Defender](#aws-iot-device-defender)
-        * [AWS IoT Over-the-air Updates (Release Candidate)](#aws-iot-over-the-air-updates-(release-candidate))
+        * [AWS IoT Over-the-air Update](#aws-iot-over-the-air-update-v2.0.0-(release-candidate))
```

Comment:
[AWS IoT Over-the-air Update](#aws-iot-over-the-air-update-v2.0.0-(release-candidate)) 
I guess the tag is without (). Please cross check hyperlinks. 

=> [AWS IoT Over-the-air Update (Release Candidate)](#aws-iot-over-the-air-update-v2.0.0-release-candidate)

Reply:
It is also without dots. This has been updated and should work now.